supervisord.conf

```diff
@@ -63,7 +63,7 @@ command =
         --config=/etc/grafana/grafana.ini
         cfg:default.paths.data=/srv/grafana
         cfg:default.paths.plugins=/var/lib/grafana/plugins
-        cfg:default.paths.logs=/srv/logs
+        cfg:default.paths.logs=/var/log/grafana
```

Comment:
Sorry, I did not notice that. :( We should use `/srv/logs` there too. And that changed should be synced with Ansible too. /cc @hors